app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (
```

Comment:
Separate your local imports and built-in library imports with one blank line according to PEP-8.